### PR TITLE
test(cdr-breadcrumb): refactor breadcrumb tests

### PR DESCRIPTION
**Pull request by KenjiCrosland**

### CODE REVIEW

File: src/components/breadcrumb/__tests__/CdrBreadcrumb.spec.js [hunk 1]
```diff
@@ -41,81 +41,96 @@ const itemsB2 = [
...
+      expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBe(false);
+      await wrapper.setProps({ items: itemsB.concat(itemsB2) });
+      await wrapper.vm.$nextTick();
+      expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBe(true);
+    });
+  })
+});
+
+
+//The active element is <body> after a click. This may be an issue with the test environment. We may need document.activeElement tests to be e2e tests --Kenji
```
Comment (KenjiCrosland) [inline src/components/breadcrumb/__tests__/CdrBreadcrumb.spec.js:133]: Can't seem to test change of focus in this environment. All tests for document.activeElement are commented out

Comment (benjag) [reply src/components/breadcrumb/__tests__/CdrBreadcrumb.spec.js:133]: Maybe try this? https://vue-test-utils.vuejs.org/api/options.html#attachto

Based on this answer here => https://stackoverflow.com/questions/65044900/why-does-document-activeelement-not-point-to-the-focused-element-in-jsdom-when-t

Comment (KenjiCrosland) [reply src/components/breadcrumb/__tests__/CdrBreadcrumb.spec.js:133]: Cool I'll check it out

Comment (KenjiCrosland) [reply src/components/breadcrumb/__tests__/CdrBreadcrumb.spec.js:133]: Yup it worked

**Review by benjag — APPROVED** [review]

**Pull request merged**